src/sbvr-api/sbvr-utils.ts

```diff
@@ -1302,6 +1305,9 @@ const runRequest = (
 			}
 			throw err;
 		})
+		.tapCatch(err => {
+			return runHooks('ERROR', request.hooks, { req, request, tx, error: err });
```

Comment:
Can you use `POSTRUN-ERROR` instead in case we need error handlers at more places in future